dev-aws/otel/otel.tf

```diff
@@ -0,0 +1,33 @@
+resource "kafka_topic" "otlp_spans" {
+  name               = "otel.otlp_spans"
+  replication_factor = 3
+  partitions         = 200
+  config = {
+    # retain 5GB on each partition
+    "retention.bytes" = "5368709120"
+    # keep data for 12 hours
```

Comment:
We lately had a situation, where the messages were not consumed by distributor after 12 hours. Should we maybe increase the retention period?

Reply:
This is dev, and we increased the number of partitions here to, so we can parallelise more. We'll see how it goes